test/python/pulse/test_timeslots.py

```diff
@@ -64,6 +87,18 @@ def test_default(self):
         self.assertEqual(Interval(1, 3), slot.interval)
         self.assertEqual(AcquireChannel(0), slot.channel)
 
+    def test_shift(self):
+        """Test shifting of Timeslot."""
+        slot_1 = Timeslot(Interval(1, 3), AcquireChannel(0))
+        slot_2 = Timeslot(Interval(1+5, 3+5), AcquireChannel(0))
+        shifted_slot = slot_1.shift(+5)
+        self.assertTrue(slot_2 == shifted_slot)
+
+    def test_representation_of_timeslot_object(self):
+        """Test representation of Timeslot object."""
+        slot = Timeslot(Interval(1, 5), AcquireChannel(0))
+        self.assertEqual(str(slot), 'Timeslot(AcquireChannel(0), (1, 5))')
```

Comment:
Could you make this a call to `repr` please and thank you 😄.

Reply:
Done! https://github.com/Qiskit/qiskit-terra/pull/2730/commits/4d746014157b7659aecf9bbfe7007082e3c1a530